pkg/receiver/gitproviderreceiver/internal/scraper/gitlabscraper/genqlient.graphql

```diff
@@ -1,51 +1,57 @@
-query getGroupProjects($fullPath: ID!) {
+query getAllGroupProjects(
+  $fullPath: ID!,
+  # @genqlient(pointer: true)
+  $after: String
+) {
   group(fullPath: $fullPath) {
-    projects {
+    projects(includeSubgroups: true, after: $after) {
+      count
+
+      pageInfo {
+        hasNextPage
+        endCursor
+      }
+
       nodes {
         name
+        fullPath
         createdAt
         lastActivityAt
       }
     }
   }
 }
 
-query getProject($fullPath: ID!) {
+query getProjectsByTopic($org: String!, $topics: [String!]) {
+  projects(searchNamespaces: true, search: $org, topics: $topics) {
+    nodes {
+      name
+      fullPath
+      createdAt
+      lastActivityAt
+    }
+  }
+}
+
+query getBranchNames($fullPath: ID!) {
   project(fullPath: $fullPath) {
-    name
-    createdAt
-    lastActivityAt
+    repository {
+      branchNames(searchPattern: "*", offset: 0, limit: 100000)
```

Comment:
Let's add a comment above this line why we chose 100,000 as the default

Reply:
I've added a comment for this, it's just not showing in this preview.